### PR TITLE
Update troubleshooting guide

### PR DESCRIPTION
**Pull request by Boy132**
Updated the "common errors".

### CODE REVIEW

File: docs/troubleshooting.mdx [hunk 1]
```diff
@@ -9,6 +9,8 @@ import TabItem from '@theme/TabItem';
 # Troubleshooting
 ## Panel errors
 
+If you see a `500 | SERVER ERROR` or `An unexpected error was encountered while processing this request` you have to check your panel logs by running the following command.
```
Comment (RMartinOscar) [inline docs/troubleshooting.mdx:12]: Just like i did first in https://github.com/pelican-dev/docs/pull/34 but you told me to move 500 lower

Comment (Boy132) [reply docs/troubleshooting.mdx:12]: No, I literally wrote `Please move the 502 error and CSRF error to the "common errors" section. The introductory text should stay.*`

**Review by RMartinOscar — COMMENTED** [review]
I don't get why would you loose intel

File: docs/troubleshooting.mdx [hunk 2]
```diff
@@ -36,17 +38,14 @@ You can set the correct permissions by running `chmod -R 755 /var/www/pelican/st
 
 ### Some common errors
 
-* `500 | SERVER ERROR` or `An unexpected error was encountered while processing this request` you have to check your panel logs by running the following command.
-* `502 Bad Gateway Error` or some code in plain text you either didn't install php-fpm or set the wrong version in your webserver vhost.
-* `sh: 1: mysql: not found`: If you are using MariaDB 11.4 or higher you'll have to restore the mysql alias by running `ln -s /usr/bin/mariadb /usr/bin/mysql`.
```
[the remark this answers is not in the document]
Comment (Boy132) [reply docs/troubleshooting.mdx:41]: There is now a dedicated mariadb driver. So the alias isn't needed anymore.

The MAC error isn't really common. And it deserves it's own section with a more detailed explanation.

**Pull request merged**